src/components/TitleTransferPanel/Actions/FinaliseEndorsement.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { AddressInput } from "./../AddressInput";
+import { AddressCta } from "./../AddressCta";
+
+interface FinaliseEndorsementProps {
+  finaliseEndorsementHolderAddress: string;
+  finaliseEndorsementBeneficiaryAddress: string;
+}
+
+export const FinaliseEndorsement = ({
+  finaliseEndorsementHolderAddress,
+  finaliseEndorsementBeneficiaryAddress
+}: FinaliseEndorsementProps) => {
+  return (
+    <>
+      <AddressInput address={finaliseEndorsementHolderAddress} setAddress={() => {}} disabled />
```

Comment:
```suggestion
      <AddressInput address={finaliseEndorsementHolderAddress} setAddress={setFinaliseEndorsementHolderAddress} disabled />
```

can do the pass through at this component first, the parent then implement this component with the `() => {}`